### ISSUE TITLE
Button doesn't appear

### ISSUE THREAD

**Issue by dvlmemnoch** [dragouf/Stash-Reviewers-Chrome-Extension#5]
Hi,
I'd love to try and use this plugin but am having trouble getting the group button to appear. I've read some of the other issues and still am unable to get the plugin to work. I see this error on every stash page I visit:
batch.js?locale=en-US:2 Uncaught Error: stash-plugin/header-notification missing aui/flag

Here's what I have:
- Atlassian Stash v3.6.1
- Chrome Version 49.0.2623.75 m
- Stash-Reviewers-Chrome-Extension-stash-server-version
- Stash is on a private network with limited access to WAN

Here's what I've done:
- Updated manifest.json by replacing two instances of <your_stash_domain_here> with my stash domain which is on a port other than 80.
- Run this command in the chrome console: WRM.require("wr!" + 'com.atlassian.stash.stash-web-api:state') which gave me this message: Downloading resources: batch.js?locale=en-US:215

Not sure if there are external javascript files that need to get downloaded for this to work, but so far have not been able to get it to work.

Please help, thanks!

**Comment by dragouf** [comment]
Hi,

Sorry I don't have this stash version to test. Please try to run this in chrome console:

`WRM.require("wr!" + 'com.atlassian.auiplugin:aui-flag')`

and then try to enter this to see if it works:

`require('aui/flag')`

if it works try to add at line 1458 of stash_page_injector.js:

```
var loadAuiFlag = jQuery.Deferred();
WRM.require("wr!" + 'com.atlassian.stash.stash-web-api:state').then(function(){
  loadAuiFlag.resolve();
});
```

and change line 1470 to:

`jQuery.when(loadRequirement, loadAuiFlag).done(function(){`

And please tell me if it works.

**Comment by dvlmemnoch** [comment]
Thanks for the respose @dragouf .

I tried modifying the stash_age_injector.js with the changes you suggested and it's still not working.

I entered the commands in the console and here's the output I received:
`WRM.require("wr!" + 'com.atlassian.auiplugin:aui-flag')
Downloading resources:
js!/s/d41d8cd98f00b204e9800998ecf8427e-CDN/en_US/cff2a57/1/5.7.19/_/download/batch/com.atlassian.auiplugin:aui-flag/com.atlassian.auiplugin:aui-flag.js!order
css!/s/86f0217b320ab0ae06b3dd3961a805a6-CDN/en_US/cff2a57/1/5.7.19/_/download/batch/com.atlassian.auiplugin:aui-flag/com.atlassian.auiplugin:aui-flag.css`

`require('aui/flag')
require('aui/flag')
J(R){if(R.hasOwnProperty("persistent")){var P;if(R.persistent){P=L.deprecate.getMessageLogger("persistent: true","5.8.0",{alternativeName:"close: never",extraInfo:"See https://ecosystem.atlas…`

I made sure to reload the extension to pick up the changes and I event restarted the browser with no luck.

**Comment by dragouf** [comment]
@dvlmemnoch which I don't understand is that it seems WRM.require download correctly all needed file to use aui/flag. It mean it should works. 

Can you try to fork my repo and commit changes you made to stash_age_injector.js. Like that I can check if you changed the file as required.

**Comment by shokmaster** [comment]
@dragouf @dvlmemnoch the correct code is:

```javascript
var loadAuiFlag = jQuery.Deferred();
    WRM.require("wr!" + 'com.atlassian.auiplugin:aui-flag').then(function(d) {
    loadAuiFlag.resolve();
});
```

I've opened a PR to fix it: #6

**Comment by dvlmemnoch** [comment]
Thank you @shokmaster that fixed it! The plugin is looking good so far, thanks @dragouf!
